scripts/ci/build_and_test.sh

```diff
@@ -17,17 +17,14 @@ SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
 source ${SCRIPT_DIR}/parse_args.sh "$@"
 
-if [[ -f ~/.cache/proxy_config ]]; then
-  source ~/.cache/proxy_config
-fi
-
 ENTRY=scripts/python/tao_build.py
 VENV_PATH=/opt/venv_disc
 BLADE_DISC_DIR=tao/python/blade_disc_tf
 
 # cleanup build cache
 (rm -rf build \
   && rm -rf tao/build \
+  && cd tao && bazel clean --expunge && cd .. \
```

Comment:
Why do we need to remove this? @Yancey1989 could you help to double-check this. I think the remote cache we used is enough for caching?

Reply:
I don't quite follow, you mean `bazel clean --expunge` is not necessary？

Reply:
Sorry. I missed something. It's ok.